docs/topics/rn-known-issues.adoc

```diff
@@ -17,23 +17,27 @@ For a complete list of all known issues, see the list of link:https://issues.red
 |Component
 |Summary
 
-|link:https://issues.redhat.com/browse/WINDUP-3138[WINDUP-3138]
-|CodeReady Workspace plugin
-|The plugin works with CodeReady Workspace 2.09, but not version 2.10.
+|link:https://issues.redhat.com/browse/WINDUP-3227[WINDUP-3227]
+|Migration - Application
+|The `IncludeTags` advanced option does not work.
 
-|link:https://issues.redhat.com/browse/WINDUP-3165[WINDUP-3165]
-|Web console on OpenShift Container Platform
-|Analysis fails when MTA custom labels are uploaded and advanced options are enabled.
+|link:https://issues.redhat.com/browse/WINDUP-3266[WINDUP-3266]
+|IntelliJ extension
+|An exception related to `getFile` intermittently occurs when running an analysis via the extension.
 
-|link:https://issues.redhat.com/browse/WINDUP-3160[WINDUP-3160]
+|link:https://issues.redhat.com/browse/WINDUP-3267[WINDUP-3267]
+|Web UI & Windup as a service (`windup-web`)
+|There is no option in the web console to navigate to or to download mavenized `pom.xml` files generated if you enable the `mavenized` option.
+
+|link:https://issues.redhat.com/browse/WINDUP-3318[WINDUP-3318]
 |Web console on OpenShift Container Platform
-|If you run an analysis on multiple applications and subsequently delete an application and rerun the analysis, the analysis hangs.
+|Analysis fails when MTA custom labels are uploaded and advanced options are enabled.
 
```

Comment:
@RichardHoch 
I see, you really insist on writing everything in passive voice! Consider rephrasing?
MTA

Reply:
Fixed.